CONTRIBUTING.md

```diff
@@ -71,10 +71,49 @@ git push -u fork my-feature-branch
 
 Now go to the project's GitHub Pull Request page and click "New pull request"
 
-
 ## Setup
 
-TODO setup with envs variable etc
+### PNPM
+
+First, you will need to have [`pnpm`](https://pnpm.io/fr/) installed on your machine.
+
+From the [installation page](https://pnpm.io/fr/installation), choose your preferred install method and run it.
+
+### NodeJS
+
+Next, you'll need to have [NodeJS](https://nodejs.org/en/) installed, with the required version. The required version is specified in the [`.npmrc`](.npmrc) root file, aswell as in every `pagakage.json` file :
+
+```json
+"engines": {
+    "node": "18.4.0"
+  },
+```
+
+Feel free to use the node-version manager of your choice (like [nvm](https://github.com/nvm-sh/nvm)), but we recommend using the integrated manager of pnpm. The [`pnpm env`](https://pnpm.io/fr/cli/env) command allows you to manage whatever node engine pnpm should use. Go into any `package.json` file and find the required node version, then enter the following command :
+
+```sh
+$ pnpm env use --global 18.4.0
+```
+
+If you enter the wrong version, or if the required node version changes, pnpm will throw errors until you make the apropriate changes.
+
+> :information_source: Using the `pnpm env` command, pnpm will resolve a separate node instance from the one you installed on your system. Note that the _pnpm_ version will be used for dependencies resolution **aswell** as for scripts.
+
+### Ory CLI
+
+For authentication, the [KubeStack](https://github.com/Kubessandra/KubeStack) uses [Ory](https://www.ory.sh/docs/welcome).
+
+In order to be run locally, Ory relies on it's CLI to create a proxy tunnel to the API.
+
+Install the CLI through the [official documentation page](https://www.ory.sh/docs/getting-started/local-development).
+
+### Environment variables.
+
+> :construction: TODO
+
+> :construction: Probably includes Ory stuff
+
+> :construction: @Kubessandra please fill this up, I could have played the guessing game or watched Twitch replays, but I'm layzy. :zzz: I'll gladly keep on working on this once I can get the project up & running
```

Comment:
I will, do that but I'm highly hesitating about using something like Doppler or Secret manager.

I will put all the env first without a secret manager and will se later if we add a secret manager
Will try to do this today

Reply:
A nice middle-ground could be using this zod script you had somewhere. 
At work we use `.ev.example` files with dummy values. It does not validate anything, but might be a simpler solution for now.

Reply:
Yes all the envs need are in the zod script

I will update the env section after your merge, maybe tomorrow